make/istio/run

```diff
@@ -6,6 +6,10 @@ set -o errexit -o nounset
 source "${GIT_ROOT}/bin/common/versions.sh"
 source "${GIT_ROOT}/make/include/defaults"
 
+# Add permissions to support Istio automatic sidecar injection.
+SC="${GIT_ROOT}/make/assets/istio"
```

Comment:
`SC` is not very clear. Maybe after moving to `make/istio/assets`, you could name this `assets="${GIT_ROOT}/make/istio/assets"`. Then the next line becomes `kubectl create -f "${assets}"`, which is more legible IMO.

Reply:
It make sense:-) I'll change to `istio_assets`in next patch.